src/codegen/intermediate_representation.c

```diff
@@ -225,6 +225,18 @@ void ir_femit_instruction
     fprint(file, "%33not %34%%%u", inst->operand->id);
     break;
 
+  case IR_ZERO_EXTEND:
+    fprint(file, "%33z.ext %34%%%u", inst->operand->id);
```

Comment:
I’d just do `zext` w/o the `.`

Reply:
`sext` :skull:

Reply:
![image](https://user-images.githubusercontent.com/74590115/223582361-3336f98e-8f17-4738-95c1-56cf5a4d7dd7.png)
:eyes:

Reply:
(That’s what LLVM calls it at least)